deployment/templates/fluentd.yaml

```diff
@@ -107,6 +107,40 @@ objects:
               value: ${OPS_CLIENT_KEY}
             - name: "OPS_CA"
               value: ${OPS_CA}
+            - name: "ES_COPY"
+              value: ${ES_COPY}
+            - name: "ES_COPY_HOST"
+              value: ${ES_COPY_HOST}
+            - name: "ES_COPY_PORT"
+              value: ${ES_COPY_PORT}
+            - name: "ES_COPY_SCHEME"
+              value: ${ES_COPY_SCHEME}
+            - name: "ES_COPY_CLIENT_CERT"
+              value: ${ES_COPY_CLIENT_CERT}
+            - name: "ES_COPY_CLIENT_KEY"
+              value: ${ES_COPY_CLIENT_KEY}
+            - name: "ES_COPY_CA"
+              value: ${ES_COPY_CA}
+            - name: "ES_COPY_USERNAME"
+              value: ${ES_COPY_USERNAME}
+            - name: "ES_COPY_PASSWORD"
+              value: ${ES_COPY_PASSWORD}
+            - name: "OPS_COPY_HOST"
```

Comment:
@ewolinetz It would simplify the deployment but what could be the usecases for needing to separate the two?  Might there be a scenario where a customer's logs are sensitive to a cluster administrator. Contrived case, but what about departments in the same organization which need some sort of authorization to work with one another for legal reasons. You could make the argument it would be OK for an admin to understand what is going on with the node, but its not acceptable for them to read app logs without approval.